File: bestia-server-commons/src/main/java/net/bestia/server/EntryActorNames.java
package net.bestia.server;

/**
 * Class contains toplevel actor names so the user can address them with
 * messages.
 * 
 * @author Thomas Felix
 *
 */
public final class EntryActorNames {

	public static final String ENTITY_MANAGER = "entity";
	
	public static final String SHARD_ENTITY = "entityShard";
	public static final String SHARD_CONNECTION = "connectionShard";
	
	private EntryActorNames() {
		// no op.
	}
}

File: bestia-server-commons/src/main/kotlin/bestia/server/EntryActorNames.kt
package bestia.server

/**
 * Class contains toplevel actor names so the user can address them with
 * messages.
 *
 * @author Thomas Felix
 */
object EntryActorNames {
  @JvmField
  val ENTITY_MANAGER = "entity"

  @JvmField
  val SHARD_ENTITY = "entityShard"

  @JvmField
  val SHARD_CONNECTION = "connectionShard"
}